bowling-ball/Game.cs

```diff
@@ -2,16 +2,63 @@
 
 namespace BowlingBall
 {
+    
     public class Game
     {
-        public void Roll(int pins)
-        {
-            throw new NotImplementedException();
+        private int[] Rolls = new int[21];
+        private int currentRoll;
+        public void Roll(int pins)
+        {
+            Rolls[currentRoll++] = pins;

```

Comment:
Use List instead of Array. As this might result in an unhandled exception if you try to add a 22nd element.